spec/features/multiwiki_assignment_spec.rb

```diff
@@ -89,7 +89,7 @@
       click_button 'Assign Articles'
       click_button 'Assign an article'
       within('#users') do
-        first('input').set('https://wikisource.org/wiki/Wp/kiu/Heyder_Cansa')
```

Comment:
Whoops, nevermind. I was just looking at the single commit, not the current whole PR. Ignore that.